tests/neg/export-in-extension.scala

```diff
@@ -0,0 +1,40 @@
+object O:
+
+  class C(x: Int):
+    def bar = x
+    def baz(y: Int) = x + y
+    val bam = x * x
+    def :: (y: Int) = x - y
+    class D
+
+  val c1 = C(1)
+
+  object O1:
+    extension (x: Int)
+      export c1.*   // error
```

Comment:
Yes, exactly. And most of these attempts would not make sense anyway since it would be weird if the export qualifier does not use the extended parameter in any way. So we restrict it to the simplest possible case.